runtime.md

```diff
@@ -67,36 +62,51 @@ Note: these operations are not specifying any command-line APIs, and the paramen
 `state <container-id>`
 
 This operation MUST generate an error if it is not provided the ID of a container.
+Attempting to query a container that does not exist MUST generate an error.
 This operation MUST return the state of a container as specified in the [State](#state) section.
 
-### Start
+### Create
 
-`start <container-id> <path-to-bundle>`
+`create <container-id> <path-to-bundle>`
 
 This operation MUST generate an error if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error.
-Using the data in `config.json`, that are in the bundle's directory, this operation MUST create a new container.
-This includes creating the relevant namespaces, resource limits, etc and configuring the appropriate capabilities for the container.
-A new process within the scope of the container MUST be created as specified by the `config.json` file otherwise an error MUST be generated.
+If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST not be created.
+Using the data in [`config.json`](config.md), this operation MUST create a new container.
+This means that all of the resources associated with the container MUST be created, however, the user-specified process MUST NOT be run at this time.
```

Comment:
I think we should echo the lifecycle wording here and say “the user-specified code, as specified by `process`, MUST NOT be run at this time”.  Especially with the idling process reference removed, saying anything here about not running a *process* (vs. not running code) seems like it could be confusing.